bundle/phases/destroy.go

```diff
@@ -15,6 +15,8 @@ func Destroy() bundle.Mutator {
 		bundle.Defer(
 			bundle.Seq(
 				terraform.StatePull(),
+				terraform.Interpolate(),
+				terraform.Write(),
```

Comment:
Could you move these above the state pull?

Reply:
To match the deploy phase.

Reply:
done